guides/introduction/community.md

```diff
@@ -1,17 +1,26 @@
 # Community
 
-## Elixir Forum
+Although the Elixir community is still rather young, its popularity is growing on quite significan pace.
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin nibh augue, suscipit a, scelerisque sed, lacinia in, mi. Cras vel lorem. Etiam pellentesque aliquet tellus. Phasellus pharetra nulla ac diam. Quisque semper justo at risus. Donec venenatis, turpis vel hendrerit interdum, dui ligula ultricies purus, sed posuere libero dui id orci. Nam congue, pede vitae dapibus aliquet, elit magna vulputate arcu, vel tempus metus leo non est. Etiam sit amet lectus quis est congue mollis. Phasellus congue lacus eget neque. Phasellus ornare, ante vitae consectetuer consequat, purus sapien ultricies dolor, et mollis pede metus eget nisi. Praesent sodales velit quis augue. Cras suscipit, urna at aliquam rhoncus, urna quam viverra nisi, in interdum massa nibh nec erat.
+### Elixir Forum
 
-## Slack
+This is a main place for every Elixir and also Erlang programming language enthusiasts! (But not only) - https://elixirforum.com
```

Comment:
This is the best place to exchange knowledge for every Elixir and Erlang languages enthusiast! https://elixirforum.com